options/locale/locale_en-US.ini

```diff
@@ -1420,7 +1420,7 @@ issues.due_date_form_remove = "Remove"
 issues.due_date_not_writer = "You need repository write access to update an issue's due date."
 issues.due_date_not_set = "No due date set."
 issues.due_date_added = "added the due date %s %s"
-issues.due_date_modified = "modified the due date to %s from %s %s"
+issues.due_date_modified = "modified the due date from %s to %s %s"
```

Comment:
```suggestion
issues.due_date_modified = "modified the due date from %[2]s to %[1]s %[3]s"
```

Reply:
One more advantage of `%[2]s to %[1]s` (keeping the original argument order in the template) is that it won't break other languages